StaticAnalyzer/views/android/VirusTotal.py

```diff
@@ -28,43 +28,44 @@ def get_report(self, file_hash):
             except:
                 print "[ERROR] VirusTotal ConnectionError, check internet connectivity"
                 return None
-    
+
             json_response = response.json()
             return json_response
-    
+
         except:
             PrintException("[ERROR] in VirusTotal get_report")
             return None
-    
+
     def upload_file(self, file_path):
         '''
         :param file_path: file path to upload
         :return: json response / None
         '''
-        try:
-            url = self.base_url + "scan"
-            files = {
-                'file': open(file_path, 'rb')
-            }
-            headers = {
-                "apikey": settings.VT_API_KEY
-            }
+        if settings.VT_UPLOAD:
```

Comment:
This check should not be inside the upload function, the function should do what it says it does,
move this check into the get_result function